doc/user/content/sql/create-source/load-generator.md

```diff
@@ -38,10 +38,6 @@ _src_name_  | The name for the source.
 **FOR ALL TABLES** | Creates subsources for all tables in the load generator.
 **EXPOSE PROGRESS AS** _progress_subsource_name_ | The name of the progress subsource for the source. If this is not specified, the subsource will be named `<src_name>_progress`. For more information, see [Monitoring source progress](#monitoring-source-progress).
 
-### `WITH` options
-
-No options yet.
-
```

Comment:
Ah, I'd left this in because `RETAIN HISTORY` should get slotted back in here ASAP: https://github.com/MaterializeInc/materialize/pull/23956.

Reply:
The most annoying part to add back in is the grammar entry, so just commented it in the `.bnf` file, rather than remove it. The second most annoying is adding the table to this section, so the value of living the header in is pretty limited.